src/components/BookForm.js

```diff
@@ -31,33 +31,28 @@ function BookForm() {
   };
 
   return (
-    <div>
+    <div className="formm">
+      <h2 className="AdBok">ADD NEW BOOK</h2>
       <form onSubmit={handleAdd}>
         <input
+          className="titleInput"
           type="text"
-          placeholder="ID"
-          value={itemId}
-          onChange={(e) => setItemId(e.target.value)}
-        />
-        <input
-          type="text"
-          placeholder="Title"
+          placeholder="Book title"
           value={title}
           onChange={(e) => setTitle(e.target.value)}
         />
-        <input
-          type="text"
-          placeholder="Author"
-          value={author}
-          onChange={(e) => setAuthor(e.target.value)}
-        />
-        <input
-          type="text"
-          placeholder="Category"
+        <select
+          className="catInput"
           value={category}
           onChange={(e) => setCategory(e.target.value)}
-        />
-        <button type="submit">
+        >
+          <option value="">Category</option>
+          <option value="option1">Fiction</option>
+          <option value="option2">Action</option>
+          <option value="option3">Romance</option>
+        </select>
```

Comment:
- [ ] Kindly consider adding the correct font to the input placholder text and to the drop down selection inorder to match the given design

Your design
![image](https://github.com/Nessrine88/Bookstore/assets/81932478/6bef1b97-e59c-49c9-9fe9-a6c4e0bdc2ee)

Given design
![image](https://github.com/Nessrine88/Bookstore/assets/81932478/14eae41b-5995-485c-8444-44c0357d5554)

Note that your dropdown menu is grey in color while the given one is white